test/compilable/traits.d

```diff
@@ -206,3 +206,33 @@ enum m20884 = "x";
 static assert(is(typeof(__traits(getMember, T20884, m20884)) == immutable(int))); // OK now
 static assert(is(          typeof(mixin("T20884." ~ m20884)) == immutable(int)));
 static assert(is(                           typeof(T20884.x) == immutable(int)));
+
+/******************************************/
+// https://issues.dlang.org/show_bug.cgi?id=20761
+
+alias Seq(T...) = T;
+
+static assert(__traits(isSame, Seq!(1, 2), Seq!(1, 2)));
+static assert(__traits(isSame, Seq!(1, 1), Seq!(1, 1)));
```

Comment:
you should add static `assert(!__traits(isSame, Seq!(1, 1), Seq!(2, 2)));` as this was really the most pathological case that was accepted before.